Updated Kubernetes links

jurajseffer opened the pull request:
The old ones are no longer valid.

alexsomesan commented:
Thanks a lot for finding and fixing this! Much appreciated.

I did a quick search for that broken URL in the codebase and it looks like it's being used in a few other places.

This is where I found it:

* website/docs/r/ingress.html.markdown
* website/docs/r/network_policy.html.markdown
* website/docs/r/role_binding.html.markdown
* website/docs/r/role.html.markdown
* kubernetes/schema_metadata.go

Would you mind fixing it there as well?

jurajseffer commented:
@alexsomesan I've searched and replaced within the whole repo and also changed the ones that were pointing to a particular commit rather than master branch.
